src/ChangeLog/ReleaseNotes/release-notes.md

```diff
@@ -0,0 +1,79 @@
+# Release Notes
+
+All notable changes are documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+
+## [1.1.6]
+
+### Changed:
+
+-  Fixed saving screenshots.
+
+## [1.1.5]
+
+### Added:
+
+-  Introduction of the USE functionality.
+-  Capture screenshots directly to clipboard of pastel-colored Jos (ready for ppt or presentation).
+
+## [1.1.4]
+
+### Added:
+
+-  Member list entries are now fully editable. Changes are also reflected in real time in the calculated Jos.
+-  Member list can now be sorted based on every property.
+
+## [1.1.3]
+
+### Changed:
+
+-  Fix: Reloading webpage does not wipe changes
+-  Fix: Alternative names do not disappear
+
+## [1.1.2]
+
+### Changed:
+
+-  Fix: Creating Jos without using diversifier algorithm now is randomized instead of producing same jos.
+
+## [1.1.1]
+
+### Added:
+
+-  Login and Register pages have relevant error messages
+
+### Changed:
+
+-  Korean name changed to Alt name.
+-  Secondary name is no longer required for member creation.
+
+## [1.1.0]
+
+### Added:
+
+-  Login and Register functionality added. Communicates with backend to make mongodb queries
+-  Saves member list and previously created Jos and other information for each account.
+
+## [1.0.3]
+
+### Changed:
+
+-  leader score now punishes with a score of nth triangular number to favor smaller number of leaders
+-  added keys to maps returning JSX elements to stop error messages in console.
+
+## [1.0.2]
+
+### Changed:
+
+-  Project More Strongly Typed
+-  Shuffling Members moved as initial step for diversifier algorithm
+
+## [1.0.1]
+
+### Added:
+
+-  Added icon.
+-  Added jo maker title for browser tab.
+
+## [1.0.0]
```

Comment:
So we actually have two release-notes.md if we duplicate it here, which means we'll have to write our updates in two different files every release. Instead, let's try to modify the path to point to the original release-notes instead and delete this file & folder.